keps/sig-cluster-lifecycle/kubeadm/20191229-kubeadm-move-out-of-kk.md

```diff
@@ -0,0 +1,703 @@
+---
+title: Moving kubeadm out of kubernetes/kubernetes
+authors:
+  - "@neolit123"
+owning-sig: sig-cluster-lifecycle
+participating-sigs:
```

Comment:
IMO testing will certainly care about this 🙃

Reply:
i would have preferred comments that SIG Testing are interested sooner.
but yes, something that the KEP possibly needs to mention is how kind is going to build kubeadm for the k/k CI jobs.
given kind is a SIG Testing project it makes sense to have the SIG in this list.